### ISSUE TITLE
Annoyance: Help page appears after every TB update

### ISSUE THREAD

**Issue by DLasher106** [cleidigh/printing-tools-ng#233]
After every Thunderbird update (started from the button in the About box), the PrintingTools NG Help page is shown in the new TB instance. This is mildly annoying.

![image](https://github.com/cleidigh/printing-tools-ng/assets/18547206/bc4c2914-286b-4fdf-be48-71e4bf51ae22)

**Comment by cleidigh** [comment]
@DLasher106 
Yup, I noticed this with an upgrade recently myself. Have to change logic to distinguish TB versus extension update. Something changed with v115.
Will Q up for a point release.
Thanks 
Christopher

**Comment by cleidigh** [comment]
@DLasher106 
I now have help only loaded on ptng updates or install.
You can get the beta here : 

- #237 

@cleidigh
